tests/base.py

```diff
@@ -75,7 +59,226 @@ def required_environment_variables(self):
                     'TAP_HELPSCOUT_CLIENT_SECRET',
                     'TAP_HELPSCOUT_CLIENT_ID'])
 
+    def expected_metadata(self):
+        """The expected streams and metadata about the streams"""
+        return {
+            "conversations": {
+                self.PRIMARY_KEYS: {"id"},
+                self.REPLICATION_METHOD: self.INCREMENTAL,
+                self.REPLICATION_KEYS: {"updated_at"}
+                #Associated to TDL - 16188
+                #self.REPLICATION_KEYS: {"user_updated_at"}
```

Comment:
This can be dropped from the base file, but a doc task should be made that explains the compounded usage of this key and also `customer_down_time` or something. Both of these are then used for bookmarking.

Reply:
done